### ISSUE TITLE
Unable to create automated build Github

### ISSUE THREAD

**Issue by Fiwoo** [docker/hub-feedback#1299]
Hi Docker team,

I'm trying to configure automated builds with our Github repositories but it seems there are internal issues. I was able to connect properly my DockerHub account and the Github account but then, when trying to create an automated build for one of our repositories, it stays in the Creating status. Checking the console, I can see the following error message:

 https://hub.docker.com/v2/repositories/github/device-management/autobuild/ 403 (FORBIDDEN)

Could you please provide some help?

Best regards

**Comment by Fiwoo** [comment]
Ok, found the hint in StackOverflow (it seems it is an old bug that is still there...)

https://stackoverflow.com/questions/42792240/dockerhub-create-automated-build-step-stuck-at-creating

The workaround is to use the following URL, using the details of your repo https://hub.docker.com/add/automated-build/github/form/<GITHUB USER ID>/<GITHUB REPO NAME>/?namespace=<DOCKERHUB NAMESPACE>

I suggest to fix it soon :)

Regards

**Comment by kallisti5** [comment]
This just started happening to me as well... you can't create automated builds for your Org anymore. This is a really nice feature and kind of the point of docker hub.

Solved it via that stackoverflow hack.

```https://hub.docker.com/add/automated-build/github/form/<GITHUB USER ID>/<GITHUB REPO NAME>/?namespace=<DOCKERHUB NAMESPACE>```

**Comment by kjeldflarup** [comment]
I tried the hack, but it just hangs in "Creating"

**Comment by salmino** [comment]
@kjeldflarup If you go to `https://hub.docker.com/add/automated-build/github/form/<GITHUB USER ID>/<GITHUB REPO NAME>/?namespace=<DOCKERHUB NAMESPACE>` via this link https://hub.docker.com/add/automated-build/github/ then the `namespace` defaults to `github`. Change `github` to your Docker Hub username and you should be golden :)

**Comment by layik** [comment]
This is still happening.

**Comment by solocommand** [comment]
Still happening, and the workaround no longer works -- throws a 404.

**Comment by TrueOsiris** [comment]
Indeed. Still happening. My old autobuilds still work. can't add new ones.
Gonna try the commandline push & see if it autobuilds after that.

**Comment by phil294** [comment]
It seems that for 14 months, it has not been possible to connect a Github repo to a Docker hub repo. Automated builds cannot be configured.

Why does this issue not have major attenention right now? What am I missing?

**Comment by timja** [comment]
I can connect some repos, but lots I can't...
Seems like old repos work but any new repos it won't find

I've emailed support and so did another person i work with 2 weeks ago

**Comment by solocommand** [comment]
It seems to be tied to the linked GitHub account. I had created a service account expressly for linking an org to docker hub. This service account can never see it’s org/repos here, but my personal GH can see everything when linked.

**Comment by nordicdyno** [comment]
same issue, github relinking works for me

**Comment by smurex** [comment]
relinking also worked for me. saved my day!

**Comment by bastelfreak** [comment]
I think I run into the same issue. My service github account pccibot was linked to the voxpupuli hub.docker.com/voxpupuli org. I wasn't able to access the voxpupuli github org. However, it works if I link my personal github account.

**Comment by kallisti5** [comment]
Same issue still happening

**Comment by kallisti5** [comment]
Issue still happening

**Comment by solocommand** [comment]
Still happening with service account
